cmd/mountv1.go

```diff
@@ -293,6 +297,8 @@ func parseFuseConfig(config []string) error {
 			bfv2AllowOtherOption = true
 		} else if v == "allow_other=false" {
 			bfv2AllowOtherOption = false
+		} else if v == "nonempty" {
```

Comment:
Should we add this to the mountv1 test as well?

Reply:
In v1 test we do not have any mount related tests its only the config file related tests, so I skipped it. Let me see if we can add a test case just to create config with this option.